package.json

```diff
@@ -36,6 +36,7 @@
   },
   "dependencies": {
     "@bitcoin-design/bitcoin-icons-react": "^0.1.9",
+    "@bitcoinerlab/secp256k1": "^1.0.5",
```

Comment:
@bumi @reneaaron I found this library which does not use WebAssembly. Updating the liquid secp256k1 interface is more difficult than I thought. This one just works.

If you guys are happy with this, we could also review if the liquid interface is even needed. Maybe this same one can be used there as well